distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+For example, an image may have different attributes than a PDF document, so each could be defined to have their own metacard type.
+
+===== Default Metacard Type and Attributes
+
+Most metacards within the system are created using the default metacard type or a metacard type based on the default type.
+The default metacard type of the system can be programmatically retrieved by calling `ddf.catalog.data.BasicTypes.BASIC_METACARD`.
+The name of the default `MetacardType` can be retrieved from `ddf.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+
+The default metacard type has the following required attributes.
+Though the following attributes are required on all metacard types, setting their values is optional except for ID.
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
+[cols="4*", options="header"]
+|===
+
+|`Metacard` Constant
+|Attribute Name
+|Attribute Format
+|Description
+
+|`CONTENT_TYPE`
+|`metadata-content-type`
+|STRING
+|Attribute name accessing for the metadata content type of a Metacard.
+
+|`CONTENT_TYPE_VERSION`
+|`metadata-content-type-version`
+|STRING
+|Attribute name for the version of the metadata content accessing type of a Metacard.
+
+|`CREATED`
+|`created`
+|DATE
+|Attribute name for the date/time *this Metacard* was created.
+
+|`EFFECTIVE`
+|`effective`
+|DATE
+|Attribute name for the date/time of *the product* represented by the Metacard.
+
+|`EXPIRATION`
+|`expiration`
+|DATE
+|Attribute name for the date/time the Metacard is no longer valid and can be removed.
+
+|`GEOGRAPHY`
+|`location`
+|GEOMETRY
+|Attribute name for the location for this Metacard.
+
+|`ID`
+|`id`
+|STRING
+|Attribute name for the ID of the Metacard.
+
+|`METADATA`
+|`metadata`
+|XML
+|Attribute name for the XML metadata for this Metacard.
+
+|`MODIFIED`
+|`modified`
+|DATE
+|Attribute name for the date/time this Metacard was last modified.
+
+|`RESOURCE_SIZE`
+|`resource-size`
+|STRING
+|Attribute name for the size in bytes of the product this Metacard represents.
+
+|`RESOURCE_URI`
+|`resource-uri`
+|STRING
+|Attribute name for the URI reference to the product this Metacard represents.
+
+|`TARGET_NAMESPACE`
+|`metadata-target-namespace`
+|STRING
+|Attribute name for the target namespace of the metadata content type of a Metacard.
+
+|`THUMBNAIL`
+|`thumbnail`
+|BINARY
+|Attribute name for the thumbnail image of the product this Metacard represents. The thumbnail must be of MIME Type `image/jpeg` and be less than 128 kilobytes. 
+
+|`TITLE`
+|`title`
+|STRING
+|Attribute name for the title of the Metacard.
+
+|===
+
+[NOTE]
+====
+It is highly recommended when referencing a default attribute name to use the `ddf.catalog.data.Metacard` constants whenever possible.
+====
+
+[WARNING]
+====
+Every <<_sources,Source>> should at the very least return an ID attribute according to Catalog API.
+Other fields may or may not be applicable, but a unique ID must be returned by a source.
+====
+
+===== Extensible Metacards
```

Comment:
Would like to see a discussion here about the other Attribute classes that are available. See other types at https://github.com/codice/ddf/tree/master/catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types; e.g. Associations, Contact, DateTime, Location, Media, Security, Topic, Validation, Security. 

Additionally, the preferred method for extending MetacardType for creating new metacard types is to implement them in blueprint. An implementation example of this should be shown here; e.g. https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-metacardtype/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L17